src/shared/tpm2-util.h

```diff
@@ -6,6 +6,11 @@
 #include "json.h"
 #include "macro.h"
 
+typedef enum TPM2Flags {
+        TPM2_FLAGS_USE_PIN = 1,
```

Comment:
we usually define flags as:

```c
typedef enum xxx {
        yyy = 1 << 0,
        zzz = 1 << 1,
        aaa = 1 << 2,
};
```

given this is a flags enum, please hence replace the right-hand side of this line with `1 << 0`, even if it's exctly the same. But it will visibly mark this as bitmask, and that's a good thing

Reply:
i documented this now in #22732